app/views/events/settings/_admin.html.erb

```diff
@@ -109,6 +109,38 @@
     <% end %>
   <% end %>
 
+  <% admin_tool do %>
+    <%= form_with(model: event, local: true, html: { "x-data" => "{ship_type: '#{event.stripe_card_shipping_type}'}" }) do |form| %>
+
+      <h3 id="sub_organizations" class="mb2 mt1">Sub-organizations</h3>
+
+      <ul>
+        <% @event.subevents.order(created_at: :desc).each do |subevent| %>
+          <li>
+            <%= link_to subevent.name, event_path(subevent) %>
+          </li>
+        <% end %>
+      </ul>
+
+      <div class="field">
+        <%= form.label :subevent_plan, "Sub-organization default plan" %>
+        <%= form.select :subevent_plan, { selected: @event.subevent_plan }, { disabled: } do %>
+          <optgroup label="🚫 Disabled">
+            <option value="">🚫 Turn off sub-organizations</option>
+          </optgroup>
```

Comment:
It's not the end of the world but I don't feel great about this versus adding a boolean to `Event::Configuration`. also both of these fields should exist in `Event::Configuration`.